notebooks/how_to/explore_test_suites.ipynb

```diff
@@ -53,7 +67,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import validmind as vm"
+    "# Replace with your code snippet\n",
+    "\n",
+    "import validmind as vm\n",
+    "\n",
+    "vm.init(\n",
+    "    api_host=\"https://api.prod.validmind.ai/api/v1/tracking\",\n",
+    "    api_key=\"...\",\n",
+    "    api_secret=\"...\",\n",
+    "    project=\"...\"\n",
+    ")"
```

Comment:
This notebook technically does not require initializing with credentials since it's only exploring available tests and not running anything. We can leave this snippet here for consistency anyway.

Reply:
Roger, saw that and was wondering about it. Will leave for now.